pocs/camera/canon_gphoto2.py

```diff
@@ -35,6 +35,8 @@ def connect(self):
 
         # Properties to be set upon init.
         prop2index = {
+            '/main/settings/datetime': 'now',             # Current datetime
```

Comment:
Note: this is untested!

I see in the manual and elsewhere this config property:
```
/main/actions/syncdatetime
Label: Synchronize camera date and time with PC
Type: TOGGLE
```
It might be another way to achieve this result.

Reply:
I saw this setting but wasn't quite clear how it worked. I toggled it but it didn't seem to update the time.

I'll add some tests. Need to get into those real hardware tests to take a look.